www/configurator/src/app/css-transpiler/css-transpiler.js

```diff
@@ -47,15 +68,18 @@ class CssTranspiler {
       }
     });
 
-    // Append the new vars to the end of our template css
-    let cssWithAppendedVars = `${this.templateCss.slice(0)} :root {`;
-    Object.keys(cssVars).forEach(cssVarKey => {
-      cssWithAppendedVars += `\n${cssVarKey}: ${cssVars[cssVarKey].current || cssVars[cssVarKey].value};`;
+    // Add the onmessage here
+    this.asyncPostcssWorker.worker.postMessage({
+      templateCss: this.templateCss.slice(0),
```

Comment:
Originally I was worred about the original reference being modified, but it is not after I made some changes.

done